packages/components/package.json

```diff
@@ -3,14 +3,14 @@
   "version": "0.0.2",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
-  "module": "dist/index.esm.js",
+  "source": "src/index.tsx",
   "license": "AGPL-3.0",
   "dependencies": {
     "html-truncate": "^1.2.2",
     "mime": "^2.5.2",
     "prop-types": "^15.7.2",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
```

Comment:
I changed the react version back to 17 because typings were messing up somewhere.